develop/dev-guide-aws-quicksight-integration.md

```diff
@@ -0,0 +1,91 @@
+
+Here are some of the benefits of using Amazon QuickSight for analytics, data visualization, and reporting:
+
+- The in-memory engine, called SPICE, responds with blazing speed.
+- No upfront costs for licenses and a low total cost of ownership (TCO).
+- Collaborative analytics with no need to install an application.
+- Combine a variety of data into one analysis.
+- Publish and share your analysis as a dashboard.
+- Control features available in a dashboard.
+- No need to manage granular database permissions—dashboard viewers can see only what you share.
+
+## Prerequisites
+
+- [TiDB Cluster](#create-tidb-cluster)
+- [Amazon QuickSight Account](https://portal.aws.amazon.com/billing/signup?client=quicksight&fid=441BE2A63D1F1F56-313F2AF2462BDF3C&redirect_url=https%3A%2F%2Fquicksight.aws.amazon.com%2Fsn%2Fconsole%2Fsignup#/start&refid=ha_awssm-evergreen-free_tier)
+
+## Create TiDB Cluster
+
+TiDB (/’taɪdiːbi:/, "Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability.
+
+<CustomContent platform="tidb">
+
+If you don't have a TiDB cluster, you can follow the [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md) to create a TiDB cluster.
+
+</CustomContent>
+
+Here is the way to create a playground TiDB cluster. In this section, create a TiDB cluster using [TiDB Cloud Dedicated Tier](https://docs.pingcap.com/tidbcloud/select-cluster-tier#dedicated-tier). But you can create your TiDB cluster anywhere. Just ensure that Amazon QuickSight can connect to your TiDB cluster.
+
+You can follow the document to [Create a TiDB Cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster), to create a TiDB Cloud Dedicated Tier cluster.
+
+## Integrate Amazon QuickSight and TiDB
+
+1. You need to [get an account](https://portal.aws.amazon.com/billing/signup?client=quicksight&fid=441BE2A63D1F1F56-313F2AF2462BDF3C&redirect_url=https%3A%2F%2Fquicksight.aws.amazon.com%2Fsn%2Fconsole%2Fsignup#/start&refid=ha_awssm-evergreen-free_tier) for [Amazon QuickSight](https://aws.amazon.com/quicksight). If you register successfully, you can see a page like this.
+
+    ![aws quicksight register](/media/develop/aws-quicksight-register.png)
+
+2. Click **Go to Amazon QuickSight**. It will jump to the [home page of Amazon QuickSight](https://quicksight.aws.amazon.com/sn/start/analyses).
+
+    ![aws quicksight home page](/media/develop/aws-quicksight-home.png)
+
+3. In the **Datasets** tab, click **New dataset**.
+
+    ![aws quicksight new dataset](/media/develop/aws-quicksight-new-dataset.png)
+
+4. And you can select the **MySQL** card. Because TiDB is highly compatible with the MySQL protocol and supports [most MySQL syntax and features](/mysql-compatibility.md), most MySQL connection libraries are compatible with TiDB.
```

Comment:
```suggestion
4. Then, select the **MySQL** card. Because TiDB is highly compatible with the MySQL protocol and supports [most MySQL syntax and features](/mysql-compatibility.md), most MySQL connection libraries are compatible with TiDB.
```